plugin.xml

```diff
@@ -32,16 +32,20 @@
         <engine name="cordova-android" version=">=6.4.0"/>
 	</engines>
 
-  <js-module src="src/www/util.js">
-      <runs/>
+  <js-module src="src/www/util.js" name="IonicWebView">
+      <clobbers target="window.Ionic.WebView"/>
```

Comment:
I think this is supposed to be `Ionic.WebView`, but it probably works because you can do `window.window.window.window.window...`